baseline/OSPS-VM.yaml

```
@@ -94,4 +94,47 @@ criteria:
       status check passes before changes can be
       merged.
     control_mappings: # TODO
+
+  - id: OSPS-VM-05
+    maturity_level: 1
+    criterion: |
+      The project publishes contacts and process
+      for reporting vulnerabilities.
+    rationale: |
+       # TODO 
+    details: |
+       Create a security.md (or similarly-named) file that contains security 
+       contacts for the project and provide project's
+      process for handling vulnerabilities in the project or dependencies.
```

```suggestion
       process for handling vulnerabilities in the project or dependencies.
```
Indentation error is breaking the build